src/net/sourceforge/kolmafia/session/RufusManager.java

```diff
@@ -553,13 +550,12 @@ public static void handleShadowRiftNC(int choice, String responseText) {
       case 1499 -> {
         // The Shadow Labyrinth
         Preferences.setInteger("shadowRiftLastNC", currentTurns);
-        lastNC = currentTurns;
+        Preferences.resetToDefault("encountersUntilSRChoice");
       }
       case 1500 -> {
         // Like a Loded Stone
         ResultProcessor.removeItem(ItemPool.RUFUS_SHADOW_LODESTONE);
       }
     }
-    Preferences.setInteger("encountersUntilSRChoice", 11 - Math.max(currentTurns - lastNC, 0));
```

Comment:
Since you no longer use lastNC, you no longer need to get shadowRiftLastNC.

This PR eliminates all "gets" of shadowRiftLastNC.
Therefore, you no longer need either of the "sets".
Therefore, you no longer need the property.